extensions-core/kafka-extraction-namespace/src/main/java/io/druid/query/lookup/KafkaLookupExtractorFactory.java

```diff
@@ -327,7 +327,7 @@ public boolean close()
   }
 
   @Override
-  public boolean replaces(@Nullable LookupExtractorFactory other)
+  public boolean equals(@Nullable Object other)
```

Comment:
I explicitly had this as replaces so as to differentiate it from the java object `equals` method. And to make it extremely specific in what the boolean value means.